util/xmlformat/README

```diff
@@ -14,10 +14,10 @@ If you installed the package xmlformat you can shorten it to:
 xmlformat -f util/xmlformat/xmlformat.conf -i xml-file
 
 To reformat all files of the english guide:
-for i in guide/C/*.xml; do util/xmlformat/xmlformat.pl -f util/xmlformat/xmlformat.conf -i $i ; done
+for i in C/guide/*.xml; do util/xmlformat/xmlformat.pl -f util/xmlformat/xmlformat.conf -i $i ; done
```

Comment:
Da das Hauptdokument  in `index.dokbook` umbenannt wurde, wird es nicht mehr erfaßt. M.E. die einfachste und sauberste Lösung: Umbennennung der restlichen .xml nach .dokbook (oder aller nach .dbk, was auch eine  offizielle Endung sein soll —erfordert Test.)

Reply:
There was @gjanssens not very consequent. ;-)

Wenn du meinst, dann lass es drüber laufen. `<revision>` wurde von uns geändert wird darin verwendet.